docs/network-flow-visibility.md

```diff
@@ -171,6 +189,118 @@ through [Antrea Agent apiserver endpoint](prometheus-integration.md):
+|-------------------------------------------|---------------|----------|-------------|
+| packetTotalCountFromSourceNode            | 56506         | 120      | unsigned64  |
+| octetTotalCountFromSourceNode             | 56506         | 121      | unsigned64  |
+| packetDeltaCountFromSourceNode            | 56506         | 122      | unsigned64  |
+| octetDeltaCountFromSourceNode             | 56506         | 123      | unsigned64  |
+| reversePacketTotalCountFromSourceNode     | 56506         | 124      | unsigned64  |
+| reverseOctetTotalCountFromSourceNode      | 56506         | 125      | unsigned64  |
+| reversePacketDeltaCountFromSourceNode     | 56506         | 126      | unsigned64  |
+| reverseOctetDeltaCountFromSourceNode      | 56506         | 127      | unsigned64  |
+| packetTotalCountFromDestinationNode       | 56506         | 128      | unsigned64  |
+| octetTotalCountFromDestinationNode        | 56506         | 129      | unsigned64  |
+| packetDeltaCountFromDestinationNode       | 56506         | 130      | unsigned64  |
+| octetDeltaCountFromDestinationNode        | 56506         | 131      | unsigned64  |
+| reversePacketTotalCountFromDestinationNode| 56506         | 132      | unsigned64  |
+| reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
+| reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
+| reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
+
+### Supported capabilities
+
+#### Storage of Flow Records
+
+Flow Aggregator stores the received flow records from Antrea Agents in a hash map,
+where the flow key is five-tuple of a network connection. Five-tuple consists of Source IP,
+Destination IP, Source Port, Destination Port and Transport protocol. Therefore,
+Flow Aggregator maintains one flow record for any given connection, and this flow
+record gets updated till the connection in the Kubernetes cluster becomes invalid.
+
+#### Correlation of Flow Records
+
+In the case of inter-Node flows, there are two flow records, one
+from the source Node, where the flow originates from, and another one from the destination
+Node, where the destination Pod resides. Both the flow records contain incomplete
+information as mentioned [here](#types-of-flows-and-associated-information). Flow
+Aggregator provides the support for the correlation of the flow records from the
+source Node and the destination Node, and it exports a single flow record with complete
+information both for inter-Node and intra-Node flows.
+
+#### Aggregation of Flow Records
+
+Flow Aggregator aggregates the flow records that belong to single connection. As part
+of aggregation, fields such as flow timestamps, flow statistics etc. are updated.
+For the purpose of updating flow statistics fields, Flow Aggregator introduces the
+[new fields](#ies-from-antrea-ie-registry) in IP Antrea Enterprise registry corresponding
```

Comment:
What is "IP Antrea Enterprise registry"?

Reply:
Typo from my side.. it is Antrea Enterpise IPFIX registry.